consensus/polybft/contractsapi/test-contracts/ZexCoinERC20.json

```diff
@@ -0,0 +1,262 @@
+{
```

Comment:
Let's remove the ZexCoinERC20 from `loadtest/contracts/ZexCoinERC20.json` and use this one in that case (only `multiple_ERC20.js` load test cenario should be affected from what I can see).

Reply:
I would leave it for now, until we remove the k6 stuff completely. This will be done in another PR.